subprojects/openvpn3/openvpn3.c

```diff
@@ -475,13 +475,14 @@ void send_auth(char *session_object, char *username, char *password)
         return;
     }
 
+    if (UniqueSession == NULL){
```

Comment:
Wouldn't it be better to move this if() statement to line 461, and assign the result from the `proxy_new_for_bus` call directly to `UniqueSession` there?

The current code flow might end up with assigning more proxy connection objects, which would end up as a memory leak (IIRC, proxy objects needs to be freed via `g_object_unref()`  to release memory consumed by the proxy connection)